test/conformance/tests/sriov_operator.go

```diff
@@ -889,7 +889,7 @@ var _ = Describe("[sriov] operator", func() {
 						}
 					}
 					return ""
-				}, 2*time.Minute, 10*time.Second).Should(ContainSubstring("Insufficient openshift.io/%s", resourceName),
+				}, 2*time.Minute, 10*time.Second).Should(ContainSubstring("failed to ensurePod %s/%s since it is not yet scheduled", namespaces.Test, runningPodB.Name),
```

Comment:
I tried reproducing the case where there is no enough VFs to schedule a pod and I get the error:

```
$ oc get events -n ...
...
32s         Warning   FailedScheduling   pod/testpod-b-7vs4m   0/5 nodes are available: 2 Insufficient openshift.io/sriovnicMultiNetworkpolicyResource, 3 node(s) had taint {node-role.kubernetes.io/master: }, that the pod didn't tolerate.
```

In which case does it return  `failed to ensurePod ...`?

Reply:
@zeeke Please check logs from my cluster. It's 4.10.12
Description of problem:
Test case failed:

```
• Failure [732.465 seconds]
[sriov] operator
/home/nkononov/go/src/github.com/k8snetworkplumbingwg/sriov-network-operator/test/conformance/tests/sriov_operator.go:66
  Generic SriovNetworkNodePolicy
  /home/nkononov/go/src/github.com/k8snetworkplumbingwg/sriov-network-operator/test/conformance/tests/sriov_operator.go:169
    Virtual Functions
    /home/nkononov/go/src/github.com/k8snetworkplumbingwg/sriov-network-operator/test/conformance/tests/sriov_operator.go:835
      should release the VFs once the pod deleted and same VFs can be used by the new created pods [It]
      /home/nkononov/go/src/github.com/k8snetworkplumbingwg/sriov-network-operator/test/conformance/tests/sriov_operator.go:837

      Timed out after 120.000s.
      Error to detect Required Event
      Expected
          <string>: failed to ensurePod sriov-conformance-testing/testpod-c6p85 since it is not yet scheduled
      to contain substring
          <string>: Insufficient openshift.io/testresource

      /home/nkononov/go/src/github.com/k8snetworkplumbingwg/sriov-network-operator/test/conformance/tests/sriov_operator.go:892

Test case name:
[sriov] operator Generic SriovNetworkNodePolicy Virtual Functions should release the VFs once the pod deleted and same VFs can be used by the new created pods
due to change in events.

Failed test case condition:

Eventually(func() string {
	events, err := clients.Events(namespaces.Test).List(context.Background(),metav1.ListOptions{})
	Expect(err).ToNot(HaveOccurred())

	for _, val := range events.Items {
		if val.InvolvedObject.Name == runningPodB.Name {
			return val.Message
		}
	}
	return ""
}, 2*time.Minute, 10*time.Second).Should(ContainSubstring("Insufficient openshift.io/%s", resourceName),"Error to detect Required Event")

Actual even messages
Events:
  Type     Reason                  Age                From               Message
  ----     ------                  ----               ----               -------
  Warning  FailedScheduling        34s                default-scheduler  0/5 nodes are available: 1 Insufficient openshift.io/testresource, 1 node(s) didn't match Pod's node affinity/selector, 3 node(s) had taint {node-role.kubernetes.io/master: }, that the pod didn't tolerate.
  Warning  FailedScheduling        19s                default-scheduler  0/5 nodes are available: 1 Insufficient openshift.io/testresource, 1 node(s) didn't match Pod's node affinity/selector, 3 node(s) had taint {node-role.kubernetes.io/master: }, that the pod didn't tolerate.
  **Warning  ErrorAddingLogicalPort  34s (x2 over 34s)  controlplane       failed to ensurePod sriov-conformance-testing/testpod-rhf74 since it is not yet scheduled**

```

Reply:
+1